.travis.yml

```diff
@@ -15,15 +15,15 @@ cache:
 language: php
 
 php:
-  - 7.2.6
+  - 7.3
```

Comment:
This should not be updated. The Docker image this runs on is 7.2.6 which would be a different PR. Please revert this.